keel-plugin/src/main/kotlin/com/netflix/spinnaker/keel/plugin/ResolvableResourceHandler.kt

```diff
@@ -173,6 +173,12 @@ interface ResolvableResourceHandler<S : Any, R : Any> : KeelPlugin {
    * @return `true` if this plugin is still busy running a previous actuation, `false` otherwise.
    */
   suspend fun actuationInProgress(name: ResourceName): Boolean = false
+
+  /**
+   * Called when a resource returns to a valid state. There is no guarantee this will only be called
+   * once. Default implementation is a no-op.
+   */
+  suspend fun onActuationComplete(resource: Resource<S>, current: R) {}
```

Comment:
Hook we can call on an handler to do stuff when a resource returns to a valid state